src/PackagerNewCommand.php

```diff
@@ -149,6 +157,9 @@ public function handle()
         $this->info('Package created successfully!');
         $this->output->newLine(2);
         $bar = null;
+
+        //composer dump-autoload to identify new MyPackageServiceProvider
+        $this->composer->dumpAutoloads();
```

Comment:
I prefer a `$this->helper->dumpAutoloads();` because the other commands also need to dump them.